OpenLisp.NET.sln

```diff
@@ -97,38 +93,6 @@ Global
 		{648A6B9E-B246-4E70-93C8-4483D0B91435}.Release|x64.Build.0 = Release|Any CPU
 		{648A6B9E-B246-4E70-93C8-4483D0B91435}.Release|x86.ActiveCfg = Release|Any CPU
 		{648A6B9E-B246-4E70-93C8-4483D0B91435}.Release|x86.Build.0 = Release|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Debug|ARM.ActiveCfg = Debug|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Debug|ARM.Build.0 = Debug|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Debug|x64.ActiveCfg = Debug|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Debug|x64.Build.0 = Debug|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Debug|x86.ActiveCfg = Debug|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Debug|x86.Build.0 = Debug|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Release|Any CPU.Build.0 = Release|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Release|ARM.ActiveCfg = Release|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Release|ARM.Build.0 = Release|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Release|x64.ActiveCfg = Release|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Release|x64.Build.0 = Release|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Release|x86.ActiveCfg = Release|Any CPU
-		{96A59125-9AE5-4E91-8741-B5214B469450}.Release|x86.Build.0 = Release|Any CPU
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Debug|Any CPU.ActiveCfg = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Debug|Any CPU.Build.0 = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Debug|ARM.ActiveCfg = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Debug|ARM.Build.0 = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Debug|x64.ActiveCfg = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Debug|x64.Build.0 = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Debug|x86.ActiveCfg = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Debug|x86.Build.0 = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Release|Any CPU.ActiveCfg = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Release|Any CPU.Build.0 = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Release|ARM.ActiveCfg = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Release|ARM.Build.0 = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Release|x64.ActiveCfg = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Release|x64.Build.0 = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Release|x86.ActiveCfg = Debug|x86
-		{31DC0EAE-9D92-42CB-9796-5B8CFB08486A}.Release|x86.Build.0 = Debug|x86
```

Comment:
These profiles are artifacts from `COSMOS` and the earlier work on making an `OpenLisp Machine`.